suite-native/atoms/src/DiscreetText.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useCallback, useMemo, useState } from 'react';
+import { LayoutChangeEvent } from 'react-native';
+
+import { useAtom, atom } from 'jotai';
+import { Blur, Canvas, Text as SkiaText, useFont } from '@shopify/react-native-skia';
+
+import { Color, TypographyStyle, typographyStylesBase } from '@trezor/theme';
+
+import { Text } from './Text';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const satoshiFont = require('../../../packages/theme/fonts/TTSatoshi-Regular.otf');
+
+type DiscreetValueProps = {
+    typography?: TypographyStyle;
+    color?: Color;
+    formattedAmount: string;
```

Comment:
why is this called `formattedAmount`? I would prefer just `text` or maybe use `children`